expander/ds/understanding-sitemap.ftd

```diff
@@ -331,7 +331,7 @@ lang: ftd
 -- ds.markdown:
 
 To know all about this feature you can checkout the video about
-[`How to create clean URLs`](/clean-urls/).
+[`How to create clean URLs`](/clean-urls/-/build/).
```

Comment:
Don't change the URL here instead go to `FASTN.ftd` file and change the url there from `/clean-urls/-/build/` to `/clean-urls/`